models/pendulum/pendulum.xml

```diff
@@ -1,15 +1,34 @@
 <mujoco model="pendulum">
-  <compiler angle="radian"/>
-  <statistic meansize="0.250205" extent="1.0008" center="0 0 -0.5"/>
+  <!-- Make some textures for the ground -->
```

Comment:
Can these features like textures instead be a part of a separate file like `pendulum_scene.xml` that includes `pendulum.xml`? For example: https://github.com/google-deepmind/mujoco_menagerie/tree/main/agility_cassie

This separates the properties of the system from properties of an environment.

Reply:
Good idea, I like it! Done.